packages/main/src/TreeItem.js

```diff
@@ -66,6 +67,38 @@ const metadata = {
 		icon: {
 			type: String,
 		},
+
+		/**
+		 * @private
+		 * @since 1.0.0-rc.8
+		 */
+		showToggleButtonEnd: {
+			type: Boolean,
+		},
+
+		/**
+		 * @private
+		 * @since 1.0.0-rc.8
+		 */
+		level: {
+			type: Integer,
+		},
+
+		/**
+		 * @private
+		 * @since 1.0.0-rc.8
+		 */
+		_collapsed: {
+			type: Boolean,
+		},
+
+		/**
+		 * @private
+		 * @since 1.0.0-rc.8
+		 */
+		_showSelectedOnSubItems: {
```

Comment:
This is very specific IMO, the tree doesn't have the notion of subitems, only level